documentation/src/docs/wire-protocol.md

```diff
@@ -102,3 +145,37 @@ After receiving the "Append Setup" reply, it performs the following:
 While this is happening, the server will be periodically sending it `DataAppended` replies acking messages. Note that there can be multiple "Appends Setup" for a given TCP connection. This allows a client to share a connection when producing to multiple Segments.
 
 A client can optimize its appending by specifying a large value in it's `AppendBlock` message, as the events inside of the block do not need to be processed individually.
+
+## Segment Attribute API
+
+| **API**      | **Description**     |
+|-------------|----------|
+|`GetSegmentAttribute`| The requested list of Segment attributes.|
+|`UpdateSegmentAttribute`|  - `newValue`: (Long). |
+| |                         - `expectedValue`: (Long). It represents |
+
+## Table Segment API
+
+| **API**      | **Description**     |
+|-------------|----------|
+|`CreateTableSegment`| To create Table Segment.|
+|`MergeTableSegment`| - `Target`: (String). This indicates the target of the Stream Segment to be Merged.|
+|                    |- `Source`: (String). This indicates the source Stream Segment that needs to be Merged.|
```

Comment:
`MergeSegment`, `SealSegment`, `TruncateSegment`, `DeleteSegment` should not be part of the Table Segment API. It can be grouped along the ReadSegment wirecommands.

Reply:
modified